charts/nri-prometheus/values.yaml

```diff
@@ -68,6 +68,14 @@ config:
   # scrape_enabled_label: "prometheus.io/scrape"
   # Wether k8s nodes needs to be labelled to be scraped or not. Defaults to false.
   require_scrape_enabled_label_for_nodes: true
+  # Wether the integraton sends metrics directly to New Relic (true) or to the Infrastructure agent (false)
+  # standalone: false
+  # List of emitters to use. Valid values include: 'stdout', 'telemetry',' infra-sdk'
+  # Each emitter outputs data in a different format. Use 'telemetry' if using 'standalone = true', 'infra-sdk' if 'standalone = false'. 'stdout' is for debug purposes
+  # emmiters:
+  #   - "infra-sdk"
```

Comment:
is it not pomi running standalone with the telemetry sdk always in k8s?

Reply:
if so i would remove this comments

Reply:
I'm not sure. Did you change that to set standalone automatically?

Reply:
ok, removed the emitters config. The code sets them based on standalone